src/application.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: loopback4-example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {RestApplication, RestServer, RestBindings} from '@loopback/rest';
+import {MySequence} from './sequence';
+
+/* tslint:disable:no-unused-variable */
+// Binding and Booter imports are required to infer types for BootMixin!
+import {BootMixin, Booter, Binding} from '@loopback/boot';
+/* tslint:enable:no-unused-variable */
+
+export class ShoppingApplication extends BootMixin(RestApplication) {
+  constructor(options?: ApplicationConfig) {
+    super(options);
+
+    // Set up the custom sequence
+    this.sequence(MySequence);
+
+    this.projectRoot = __dirname;
+    // Customize @loopback/boot Booter Conventions here
+    this.bootOptions = {
+      controllers: {
+        // Customize ControllerBooter Conventions here
+        dirs: ['controllers'],
+        extensions: ['.controller.js'],
+        nested: true,
+      },
+    };
+  }
+
+  async start() {
+    await super.start();
+
+    const server = await this.getServer(RestServer);
+    const port = await server.get(RestBindings.PORT);
+    console.log(`Server is running at http://127.0.0.1:${port}`);
+    console.log(`Try http://127.0.0.1:${port}/ping`);
```

Comment:
IIRC, this template has been updated recently to use `this.restServer.url`, see https://github.com/strongloop/loopback-next/pull/1572. Could you please update the implementation of `start` method accordingly? Feel free to do so in a follow-up pull request, I think it will be easier to review.